crates/hyperswitch_domain_models/src/router_flow_types/payments.rs

```diff
@@ -57,7 +57,7 @@ pub struct CalculateTax;
 pub struct SdkSessionUpdate;
 
 #[derive(Debug, Clone)]
-pub struct CreateIntent;
+pub struct Intent;
```

Comment:
Is this OK or should this be `PaymentsIntent` instead? Or we need separate `CreateIntent` and `GetIntent`?

Reply:
These can be `PaymentCreateIntent` and `PaymentGetIntent`